test/services/fileStorage/utils/copyCourseFiles.test.js

```diff
@@ -0,0 +1,86 @@
+const { expect } = require('chai');
+const appPromise = require('../../../../src/app');
+const { setupNestServices, closeNestServices } = require('../../../utils/setup.nest.services');
+const testObjects = require('../../helpers/testObjects')(appPromise());
+const { copyCourseFile } = require('../../../../src/services/fileStorage/utils/copyCourseFiles');
+
+describe('copy course files', () => {
+	let app;
+	let server;
+	let nestServices;
+
+	before(async () => {
+		app = await appPromise();
+		server = await app.listen(0);
+		nestServices = await setupNestServices(app);
+	});
+
+	after(async () => {
+		await testObjects.cleanup();
+		await server.close();
+		await closeNestServices(nestServices);
+	});
+
+	it('should copy file into course', async () => {
+		const { _id: schoolId } = await testObjects.createTestSchool({});
+		const teacher = await testObjects.createTestUser({ roles: ['teacher'], schoolId });
+		const course = await testObjects.createTestCourse();
+		const storageProvider = await testObjects.createTestStorageProvider();
+
+		const file = await testObjects.createTestFile({
+			owner: teacher._id,
```

Comment:
two times the same code that is passed to createTestFile the creating can be outsource.

Reply:
fixed